### ISSUE TITLE
Error reported when importing ResNet trained by Keras

### ISSUE THREAD

**Issue by hdd67** [deeplearning4j/deeplearning4j#9966]
--Version
every version
--Message
 -in 1.0.0-M2
 [main] ERROR org.deeplearning4j.nn.modelimport.keras.Hdf5Archive - 
java.lang.ArrayStoreException: sun.reflect.annotation.TypeNotPresentExceptionProxy
	at sun.reflect.annotation.AnnotationParser.parseClassArray(AnnotationParser.java:724)
	at sun.reflect.annotation.AnnotationParser.parseArray(AnnotationParser.java:531)
	at sun.reflect.annotation.AnnotationParser.parseMemberValue(AnnotationParser.java:355)
	at sun.reflect.annotation.AnnotationParser.parseAnnotation2(AnnotationParser.java:286)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations2(AnnotationParser.java:120)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations(AnnotationParser.java:72)
	at java.lang.Class.createAnnotationData(Class.java:3521)
	at java.lang.Class.annotationData(Class.java:3510)
	at java.lang.Class.createAnnotationData(Class.java:3526)
	at java.lang.Class.annotationData(Class.java:3510)
	at java.lang.Class.getAnnotation(Class.java:3415)
	at java.lang.reflect.AnnotatedElement.isAnnotationPresent(AnnotatedElement.java:258)
	at java.lang.Class.isAnnotationPresent(Class.java:3425)
	at org.bytedeco.javacpp.Loader.checkPlatform(Loader.java:851)
	at org.bytedeco.javacpp.Loader.load(Loader.java:971)
	at org.bytedeco.javacpp.Loader.load(Loader.java:951)
	at org.deeplearning4j.nn.modelimport.keras.Hdf5Archive.<clinit>(Hdf5Archive.java:56)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.modelHdf5Filename(KerasModelBuilder.java:229)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:149)
	at com.zh.safedriveserver.model.ReadModel.getVGG19(ReadModel.java:29)
	at com.zh.safedriveserver.model.Eval.main(Eval.java:22)
Exception in thread "main" java.lang.ExceptionInInitializerError
	at org.deeplearning4j.nn.modelimport.keras.Hdf5Archive.<clinit>(Hdf5Archive.java:63)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.modelHdf5Filename(KerasModelBuilder.java:229)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:149)
	at com.zh.safedriveserver.model.ReadModel.getVGG19(ReadModel.java:29)
	at com.zh.safedriveserver.model.Eval.main(Eval.java:22)
Caused by: java.lang.ArrayStoreException: sun.reflect.annotation.TypeNotPresentExceptionProxy
	at sun.reflect.annotation.AnnotationParser.parseClassArray(AnnotationParser.java:724)
	at sun.reflect.annotation.AnnotationParser.parseArray(AnnotationParser.java:531)
	at sun.reflect.annotation.AnnotationParser.parseMemberValue(AnnotationParser.java:355)
	at sun.reflect.annotation.AnnotationParser.parseAnnotation2(AnnotationParser.java:286)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations2(AnnotationParser.java:120)
	at sun.reflect.annotation.AnnotationParser.parseAnnotations(AnnotationParser.java:72)
	at java.lang.Class.createAnnotationData(Class.java:3521)
	at java.lang.Class.annotationData(Class.java:3510)
	at java.lang.Class.getAnnotation(Class.java:3415)
	at org.bytedeco.javacpp.Loader.checkPlatform(Loader.java:872)
	at org.bytedeco.javacpp.Loader.load(Loader.java:971)
	at org.bytedeco.javacpp.Loader.load(Loader.java:935)
	at org.bytedeco.hdf5.IdComponent.<clinit>(IdComponent.java:23)
	... 5 more
 -in 1.0.0-beta3
   main] INFO org.nd4j.versioncheck.VersionCheck - org.nd4j : nd4j-native : 1.0.0-beta3
[main] INFO org.nd4j.nativeblas.NativeOpsHolder - Number of threads used for NativeOps: 6
[main] INFO org.nd4j.nativeblas.Nd4jBlas - Number of threads used for BLAS: 6
[main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Backend used: [CPU]; OS: [Windows 10]
[main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Cores: [12]; Memory: [7.1GB];
[main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Blas vendor: [MKL]
Exception in thread "main" java.lang.ClassCastException: java.util.ArrayList cannot be cast to java.lang.Integer
	at org.deeplearning4j.nn.modelimport.keras.layers.normalization.KerasBatchNormalization.getBatchNormAxis(KerasBatchNormalization.java:354)
	at org.deeplearning4j.nn.modelimport.keras.layers.normalization.KerasBatchNormalization.<init>(KerasBatchNormalization.java:119)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:259)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:218)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:96)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:307)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:127)
	at com.zh.safedriveserver.model.ReadModel.getVGG19(ReadModel.java:29)
	at com.zh.safedriveserver.model.Eval.main(Eval.java:22)

**Comment by agibsonccc** [comment]
Closing this. You are mixing versions. We explicitly say not to do that. Use the latest stable version and do not mix.

**Comment by hdd67** [comment]
The same issue occurs in every version, which I have tried multiple times.

**Comment by agibsonccc** [comment]
Do you mind posting your pom.xml with the latest version and a reproducer?  If so I will take a look. Jfyi this module is deprecated though. I will be forcing users to use omnihub going forward which handles dl4j models and the much more flexible samediff framework. Thanks.
